x/programs/examples/pokemon.go

```diff
@@ -35,10 +35,15 @@ type Pokemon struct {
 
 func (t *Pokemon) Run(ctx context.Context) error {
 	meter := runtime.NewMeter(t.log, t.maxFee, t.costMap)
-	db := utils.NewTestDB()
-	store := newProgramStorage(db)
+	db, _, err := pebble.New(DBPath, pebble.NewDefaultConfig())
```

Comment:
is it ok that this is shared across examples? is prefixing handled somehow?

Reply:
yea, there is a program counter stored in the db. when a program is "published" we increment and return the counter as the id.